src/ui/buttons/menu.jsx

```diff
@@ -12,6 +12,9 @@ export function MenuButton({ color = TEXT_COLOR.BLACK } : {| color? : $Values<ty
     return (
         <Style css={ css }>
             <div
+                role="button"
+                // localize this, then it can be released
+                // aria-label="funding-options dropdown"
```

Comment:
Looks good 👍 -- we should have this use `props.content` then pass in the content mapping from spb when we render this menu component.

Reply:
Yeah, but I don't think I have time to add new content, so I just commented it out